pandas_vet/__init__.py

```diff
@@ -18,10 +18,22 @@ class Visitor(ast.NodeVisitor):
     errors = attr.ib(default=attr.Factory(list))
 
     def visit_Import(self, node):
+        """ 
+        Called for `import ..` and `import .. as ..` nodes.
+        """
         self.generic_visit(node)  # continue checking children
         self.errors.extend(check_import_name(node))
 
+    def visit_ImportFrom(self, node):
```

Comment:
Did you mean to include this? Right now it only has a `generic_visit`, no specific checks.

Reply:
No, it was just documenting as I was learning about.